.github/workflows/_rocm-test.yml

```diff
@@ -130,7 +130,7 @@ jobs:
           TEST_CONFIG: ${{ matrix.config }}
           SHARD_NUMBER: ${{ matrix.shard }}
           NUM_TEST_SHARDS: ${{ matrix.num_shards }}
-          PR_BODY: ${{ github.event.pull_request.body }}
+          REENABLED_ISSUES: ${{ github.event.pull_request.body }}
```

Comment:
This looks wrong as other workflow has `REENABLED_ISSUES: ${{ steps.keep-going.outputs.reenabled-issues }}`